src/illumidesk/authenticators/authenticator.py

```diff
@@ -111,6 +113,11 @@ async def authenticate(self, handler, data=None):
                 elif 'lis_person_sourcedid' in args and args['lis_person_sourcedid'] is not None:
                     username = args['lis_person_sourcedid']
                     self.log.debug('using lis_person_sourcedid for username')
+
+                # GRADES-SENDER >>>> retrieve assignment_name from custom property
+                assignment_name = (
```

Comment:
How would you feel if we get the assignment name from the `resource_link_id` argument by default?

Reply:
Resolved, I see now how the resource_link_id is the default. However, we aren't checking to see if the field is included or not. If it's not included (unlikely but possible since it's not required) then we need to handle that exception.